core-services/prow/02_config/openshift/cincinnati-graph-data/_prowconfig.yaml

```diff
@@ -9,6 +9,15 @@ tide:
     - needs-rebase
     repos:
     - openshift/cincinnati-graph-data
+  - author: openshift-bot
```

Comment:
Do we want to add this to [`trusted-apps`][1] too?

[1]: https://github.com/openshift/release/blob/6eaf4546cfeed7cf6970ff7ffa8783ed43a598ab/core-services/prow/02_config/openshift/cincinnati-graph-data/_pluginconfig.yaml#L12-L14

Reply:
It's not an app, it's an account: https://github.com/openshift-bot

Apps have a little `bot` label in GH UI:

![image](https://github.com/openshift/release/assets/712614/2fe8e54d-1434-4540-93b6-bbac9dc9a28e)

Reply:
So it should not be necessary. `trusted-apps` control whose PRs can trigger CI jobs without `/ok-to-test`, which is normally controlled by org membership. apps are not accounts, so they cannot be org members, hence `trusted-apps` was invented.